packages/platform-ios/native_modules.rb

```diff
@@ -54,6 +60,16 @@ def use_native_modules!(config = nil)
 
     spec = Pod::Specification.from_file(podspec_path)
 
+    # Skip pods that do not support the platform of the current target.
+    if platform = current_target_definition.platform
+      next unless spec.supported_on_platform?(platform.name)
```

Comment:
This is the actual semantic change of the PR, so tiny 😂

Reply:
Thanks for that comment! And also, I am impressed by the CocoaPods API :D